recipe/index.html

```diff
@@ -55,7 +55,7 @@ <h2> Instructions </h2>
       
 
         <div class="links">
-          <h2> Similar Recipes </h2>
+          <h2> Other Recipes  </h2>
```

Comment:
Actually, please make this Different recipes

Reply:
Done!